src/Search/Search.js

```diff
@@ -31,8 +31,8 @@ function WikipediaSearch() {
       srlimit: 20,
       srsearch: searchTerm,
     };
-
-    await fetch(url + new URLSearchParams(params).toString())
+  
```

Comment:
The await was added because it needs to wait to do the loading, that may be the cause of the bug you experienced? ill add it back locally and see if that fixes the issue. shouldn't be a big fix